build.gradle

```
@@ -19,7 +19,7 @@ dependencies {
 }
 
 intellij {
-    version '2019.3'
+    version 'LATEST-EAP-SNAPSHOT'
```

```suggestion
    version '2020.1'
```

The other option is to continue incrementally adding support